src/Services/TestimonialsApiService.js

```diff
@@ -0,0 +1,27 @@
+import {getRequest, postRequest, putRequest, deleteRequest} from '../../Services/publicApiService';
+const endPoint = process.env.REACT_APP_API_TESTIMONIALS
```

Comment:
Faltó el segundo punto, mala mía por no dejarlo en otro comentario bien definido.

Endpoint es una sola palabra, no es necesario el camel case:

```suggestion
const endpoint = process.env.REACT_APP_API_TESTIMONIALS
```

Corregirlo acá, y en los lugares donde después se usa!

Reply:
Jaja Sí! lo había entendido, solo que acepté el cambio desde el celu. Ahi entro a la compu y corrijo bien a todo el archivo.